src/transaction/log_lsa.hpp

```diff
@@ -51,6 +51,7 @@ struct log_lsa
 
   inline log_lsa (const int64_t value);
   inline log_lsa (const log_lsa &olsa) = default;
+  inline log_lsa (log_lsa &&olsa) = default;
```

Comment:
I thought this might be needed so that 
```
log_Gl.m_metainfo.add_checkpoint_info (trantable_checkpoint_lsa, std::move (trantable_checkpoint_info));
```
actually moves but it might not be so.